compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -130,18 +131,22 @@ static bool isSmallerThan(ArrayRef<int64_t> sourceShape,
 /// `dimSize`.
 static std::pair<OpFoldResult, OpFoldResult>
 getScaledSizeAndOffset(OpBuilder &builder, Location loc, OpFoldResult size,
-                       OpFoldResult offset, OpFoldResult dimSize,
+                       OpFoldResult offset, std::optional<OpFoldResult> dimSize,
```

Comment:
The comment doesnt match what seems to be happening here. Does it need to be updated?

Reply:
Yeah, I'll update it. Thanks for catching